src/Bundle/DependencyInjection/SyliusGridExtension.php

```diff
@@ -60,6 +62,14 @@ public function load(array $configs, ContainerBuilder $container): void
             ->addTag('sylius.grid')
         ;
 
+        $container->registerAttributeForAutoconfiguration(AsFilter::class, static function (ChildDefinition $definition, AsFilter $attribute, \ReflectionClass $reflector) {
+            $tagAttributes = [
+                'type' => $attribute->type,
```

Comment:
```suggestion
                'type' => $attribute->type ?? $reflector->getName(),
```

Reply:
The FQCN is now the default type, I used it here https://github.com/Sylius/SyliusGridBundle/pull/348/files#diff-77aaeac60781f223afad001a87631ddf56d09651d5263d8b629f58c4d7354996R36

Reply:
The doc and tests uses the new FQCN behaviour